File: prisma/fix-backticks.py
content = open('prisma/seed-llm-terms.ts', 'r', encoding='utf-8').read()
fixed = content.replace('```', r'\`\`\`')
open('prisma/seed-llm-terms.ts', 'w', encoding='utf-8').write(fixed)
print('done, replacements:', content.count('```'))

File: prisma/fix-backticks.js
const fs = require('fs');
const content = fs.readFileSync('prisma/seed-llm-terms.ts', 'utf8');
// Replace ``` with \`\`\` (escaped backticks for TypeScript template literals)
const fixed = content.split('```').join('\\`\\`\\`');
fs.writeFileSync('prisma/seed-llm-terms.ts', fixed, 'utf8');
console.log('done, replacements:', (content.match(/```/g) || []).length);
